cranelift-codegen/meta/src/shared/instructions.rs

```diff
@@ -2071,6 +2081,22 @@ pub(crate) fn define(
         .can_trap(true),
     );
 
+    let a = &Operand::new("a", IxN);
+    let x = &Operand::new("x", IxN);
+    let y = &Operand::new("y", IxN);
+
+    ig.push(
+        Inst::new(
+            "avgr",
```

Comment:
Should we use the more current abbreviation "avg", or just be explicit and call it "average" (it's not very long)? This is the first time I see this midway abbreviation for average, but I'm not the English native here.

Reply:
The complication here is that the semantics are actually "rounding average" instead of just "average"; I went along with the term ["avgr" used by the spec](fhttps://github.com/WebAssembly/simd/blob/master/proposals/simd/SIMD.md#lane-wise-integer-minimum). Do you want to expand it to "rounding_average" or "average_round"?

Reply:
Good point! Maybe then `avg_round` is a nice tradeoff? "avg" is standard for average, and we make the rounding part very explicit.